drivers/usb/usbhub/usbhub.inf

```diff
@@ -1,4 +1,6 @@
-;USB.INF
+; usbhub.sys
```

Comment:
Please don't!! This is breaking WIndows compatibility.
When you have a INF file that covers "many" drivers, this means that this set of drivers are part of "one" module: here for example the set of USB drivers.
So instead you should keep the whole USB.INF file as it is and add the "add_driver_inf" command in the main drivers/usb/ cmakelists file.

Reply:
The current separation to `usb.inf`, `usbport.inf` and `usbstor.inf` does not make much sense either - these drivers don't work without each other.